Gemfile

```diff
@@ -5,7 +5,7 @@ require File.join(Bundler::Plugin.index.load_paths("bundler-inject")[0], "bundle
 
 gem "activesupport", '~> 5.2.4.3'
 gem "ansible_tower_client", "~> 0.21.0"
-gem "cloudwatchlogger",   "~> 0.2"
+gem "cloudwatchlogger",   "~> 0.2.1"
```

Comment:
I thought we had allowed movement to the newest version of the gem for a specific reason.  @lindgrenj6 is that correct? Or can we downgrade the gem without issue.

Reply:
The 0.3.0 version was released week ago, long time after this line was updated before